CONTRIBUTING.md

```diff
@@ -187,10 +187,12 @@ follow the instructions in this section.
     1. [Chart version](helm/secrets-provider/Chart.yaml)
     1. [Default deployed version](helm/secrets-provider/values.yaml)
     
-1. Create a helm package by running the following command from the repo root: `pushd helm/secrets-provider/packages && helm package .. && popd`
-1. Create a PR for [helm-charts](https://github.com/cyberark/helm-charts) and do the following:
-    1. Add the packaged file to the *docs* folder
-    1. Execute the `reindex.sh` script file from the `helm-charts` repo root folder
+1. Create a helm package by running the following command from the repo root: `helm package helm/secrets-provider`.
+   The helm package will be in the current folder, in the format `secrets-provider-<version>.tgz`.
```

Comment:
The Helm package will be saved to the current folder and will resemble `secrets-provider-<version>.tgz`